skytemple_files/common/ppmdu_config/pmdsky_debug/extras.py

```diff
@@ -106,7 +106,12 @@ class ExtraAllSymbolsProtocol(Protocol):
 class ExtraEuArm9Functions:
     # TODO: Is this really the same as DebugPrint0 for EU???
     DebugPrint2 = Symbol(
-        [0x200C284 - EuArm9Section.loadaddress], [0x200C284], None, DEBUG_PRINT2_DESC
+        [0x200C284 - EuArm9Section.loadaddress],
+        [0x200C284],
+        None,
+        "DebugPrint2",
+        DEBUG_PRINT2_DESC,
+        "",
```

Comment:
```suggestion
        None,
```
As stated in the description of `Symbol`: "C type of this symbol. Empty string if unknown. None for functions."